aleph/manage.py

```diff
@@ -499,9 +500,12 @@ def publish(foreign_id):
 @cli.command()
 def upgrade():
     """Create or upgrade the search index and database."""
-    upgrade_system()
-    # update_roles()
-    # upgrade_collections()
+    try:
+        upgrade_system()
+        # update_roles()
+        # upgrade_collections()
```

Comment:
```suggestion
        update_roles()
        upgrade_collections()
```

Reply:
These two lines were commented 4 years ago ([here](https://github.com/alephdata/aleph/commit/0350dabefedda6af8390ccf7574227118208bacf)). Do we want to bring them back?